example/curlz/README.md

```diff
@@ -0,0 +1,119 @@
+# Overview
+This example is a zitified cURL (cURLz) example. In this example, a call will be made to a non-zitified endpoint which 
+will be utilizing an edge router to offload data. A call to a zitified endpoint is also included in this example.
+
+This example demonstrates:
+* Binding a service
+
+# Requirements
+* go 1.19 or later
+* an OpenZiti network. If you do not have one, you can use one of the [quickstarts](https://openziti.github.io/ziti/quickstarts/quickstart-overview.html) to set one up.
+* OpenZiti CLI to create services and identities on the OpenZiti Network
+
+# Build the example
+```
+cd <repo-root-dir>/sdk-golang/example/curlz/
+go build curlz.go
+```
+
+# Setup a cURLz to a non-zitified endpoint
+These steps will configure the service using the OpenZiti CLI. At the end of these steps you will have created:
+* a service called `web.endpoint`
+* an identity to connect to (dial) the service
+* the service config to connect the service to the overlay
+* the service policies required to authorize the identities for bind and dial
+
+Steps:
+1. log into Ziti. The host:port and username/password will vary depending on your network.
+
+       ziti edge login localhost:1280 -u admin -p admin
+1. Determine your edge router's name and populate this environment variable with it.
+
+       ziti edge list edge-routers
+       export ZITI_EDGE_ROUTER=<name-of-edge-router>
+1. Run this script to create everything you need.
+
+       echo Create the service config
+       ziti edge create config web.endpoint.hostv1 host.v1 '{"protocol":"tcp", "address":"www.google.com","port":'443'}'
+
+       echo Create the service
+       ziti edge create service web.endpoint --configs "web.endpoint.hostv1"
+       
+       echo Create an identity to make the dial request and enroll it
+       ziti edge create identity user curlz -a clients -o curlz.jwt
+       ziti edge enroll --jwt curlz.jwt
+       
+       echo Create service policies
+       ziti edge create service-policy web.endpoint.dial Dial --service-roles "@web.endpoint" --identity-roles "#clients"
+       ziti edge create service-policy web.endpoint.bind Bind --service-roles "@web.endpoint" --identity-roles "@${ZITI_EDGE_ROUTER}"
+       
+       echo Create edge router policies
+       ziti edge create edge-router-policy curlz-edge-router-policy --edge-router-roles '#all' --identity-roles '#clients,#servers'
+       ziti edge create service-edge-router-policy curlz-service-edge-router-policy --edge-router-roles '#all' --service-roles '@web.endpoint'
+       
+       echo Run policy advisor to check
+       ziti edge policy-advisor services
+1. Run the cURLz example for `web.endpoint`
+
+       ./curlz https://web.endpoint curlz.json
```

Comment:
you're only running `curlz` one time. what good was it to mention the 'non-ziti' service? Shouldn't you be making a curlz to https://www.google.com too?

Reply:
ooooh.. i get what you're going for now. hrmmm i just didn't get what it was all about at first here...